manager/integration/tests/common.py

```diff
@@ -2590,8 +2655,37 @@ def cleanup_node_disks(client, node_name):
         disk.allowScheduling = False
     update_disks = get_update_disks(disks)
     node = client.by_id_node(node_name)
-    node.diskUpdate(disks=update_disks)
-    node.diskUpdate(disks={})
+
+    # Retry if "too many retries error" happened.
```

Comment:
I think this doesn't belong to 1.2, because it's part of https://github.com/longhorn/longhorn/issues/685. 

You can check the commit history to know which part comes.

Reply:
Hi @innobead :
After I checked the commit history, this doesn't belong to 1.2.
I find out that my fixed is included in this [PR](https://github.com/longhorn/longhorn-tests/commit/f0e8257fd9f2a62e03c37b7bc54eebb93f8c5b28).
The `test_migration_with_unscheduled_replica` is passed, should I close my PR?

Reply:
Ok, then we don't need this anymore. Let's close this.